dapp/src/components/AccountStatusDropdown.js

```diff
@@ -60,15 +63,26 @@ const AccountStatusDropdown = ({ className, showLogin, dapp }) => {
           )}
           {/* What causes !active && account? */}
           {dapp && !active && account && <div className="dot" />}
-          {active && !correctNetwork && (
+          {active && overrideAccount && (
+            <>
+              <div className="dot white" />
+              <div className="address">
+                {`${fbt('readonly', 'readonly')}: ${overrideAccount.substring(
```

Comment:
because there is a `readonly` text, the space is not enough
![image](https://user-images.githubusercontent.com/51706089/197745317-82935f13-8ab6-4e9a-a14a-bd19a393ef9d.png)

maybe we can pass a params to the function and make it look like this
![image](https://user-images.githubusercontent.com/51706089/197745670-e572bb10-efc8-4348-80b6-4bb000bfbd93.png)

Reply:
Ah, gotcha